content/en/user-manual/graphics/lighting/shadows.md

```diff
@@ -18,9 +18,23 @@ By default, shadow casting is disabled in PlayCanvas. You have to explicitly ena
 
 ![Model Component][6]
 
-Now you need to specify which graphical objects in your scene cast and receive shadows. By default, all model components receive shadows but do not cast them. Therefore, in order to see an entity cast shadows, select it in the Hierarchy, locate the model component in the Inspector and check the 'Cast Shadows' option.
+Now you need to specify which graphical objects in your scene cast and receive shadows. By default, all render and model components cast and receive shadows. To modify these properties, select the entity in the Hierarchy, locate the render or model component in the Inspector and uncheck the 'Cast Shadows' or 'Receive Shadows' option as required.
 
-Shadows should now be visible in the Editor's Viewport.
+## Shadow Cascades
+
+When a directional shadow is used over a large area, it often exhibits aliasing, where a shadow near the camera has a low resolution. Capturing the shadow in a single shadow map requires very high and impractical resolution to improve this.
+
+Shadow cascades help to fix this problem by splitting the camera view frustum along the viewing direction, and a separate shadow map is used for each split. This gives nearby objects one shadow map, and another shadow map captures everything in the distance, and optionally additional shadow maps in between.
+
+The following properties can be used to set up shadow cascades.
+
+### Number of cascades
+
+Number of cascades represents the number of view frustum subdivisions, and can be 1, 2, 3 or 4. The default value of 1 represents a single shadow map.
+
+### Distribution of cascades
+
+The distribution of subdivision of the camera frustum for individual shadow cascades. A value in the range of 0 to 1 can be specified. Value of 0 represents a linear distribution, value of 1 represents a logarithmic distribution. Larger value increases the resolution of the shadows in the near distance.
```

Comment:
Why should a user use a smaller value if a larger value implies better results of increased resolution?

Would screenshots of the different values help here?

Reply:
This is hard to show using a screenshot. I'm updating text to explain better.